control_panel_api/permissions.py

```diff
@@ -34,7 +34,7 @@ class K8sPermissions(BasePermission):
     ]
 
     def has_permission(self, request, view):
-        if not request.user or request.user.is_anonymous():
```

Comment:
What happened with `is_anonymous`? Actually I looked it up and it's an attribute from Django 2.0 👍